lib/game_factory.rb

```diff
@@ -0,0 +1,23 @@
+class GameFactory
+  attr_reader :games
+
+  def initialize
+    @games = []
+  end
+
+  def create_games(path)
+    CSV.foreach((path), headers: true, header_converters: :symbol) do |row|
+      @games << Game.new(
```

Comment:
I like how you did this, I setup all of the info for the GameTeam object in a hash, then passed that hash in to the GameTeam.new(). I can refactor if this is how we'd like to have it across all factories.